packages/docs/page-config/ui-elements/aspect-ratio/index.ts

```diff
@@ -0,0 +1,23 @@
+import apiOptions from "./api-options";
+
+export default definePageConfig({
+  blocks: [
+    block.title("aspectRatio.title"),
+    block.paragraph("aspectRatio.summaryText"),
+
+    block.subtitle("all.examples"),
+
+    block.headline("aspectRatio.examples.default.title"),
+    block.example("Default", { hideTitle: true }),
```

Comment:
Example block will render titles automatically, if not, you have a wrong structure in locales file.

```suggestion
    block.example("Default"),
```